test/app.test.js

```diff
@@ -126,6 +127,22 @@ describe('loopback:app generator', function() {
     });
   });
 
+  it('scaffolds 3.x app when option.loopbackVersion is 3.x',
```

Comment:
Please add a test for `loopbackVersion: '2.x'` too, so that when we switch the default to 3.x, we will have a test for a non-default value around.